runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -767,6 +770,36 @@ fn wasm_instruction(ctx: &mut EstimatorContext) -> GasCost {
     per_instruction
 }
 
+fn wasm_instruction_v2(ctx: &mut EstimatorContext) -> GasCost {
+    let runner = ctx.raw_runtime();
+    let repeats = ctx.config.iter_per_block as u64;
+    let warmup_repeats = ctx.config.warmup_iters_per_block as u64;
+
+    let div = binary_op_cost(&runner, repeats, warmup_repeats, "i64", "div_s");
+    let rem = binary_op_cost(&runner, repeats, warmup_repeats, "i64", "rem_s");
+    let add = binary_op_cost(&runner, repeats, warmup_repeats, "i64", "add");
+    let mul = binary_op_cost(&runner, repeats, warmup_repeats, "i64", "mul");
+
+    let f_div = binary_op_cost(&runner, repeats, warmup_repeats, "f64", "div");
+    let f_add = binary_op_cost(&runner, repeats, warmup_repeats, "f64", "add");
+    let f_mul = binary_op_cost(&runner, repeats, warmup_repeats, "f64", "mul");
+    let f_sqrt = unary_op_cost(&runner, repeats, warmup_repeats, "f64", "sqrt");
+
+    if ctx.config.debug_wasm_op {
+        eprintln!("i64 div {div:?}");
+        eprintln!("i64 rem {rem:?}");
+        eprintln!("i64 add {add:?}");
+        eprintln!("i64 mul {mul:?}");
+
+        eprintln!("f64 div {f_div:?}");
+        eprintln!("f64 add {f_add:?}");
+        eprintln!("f64 mul {f_mul:?}");
+        eprintln!("f64 sqrt {f_sqrt:?}");
+    }
+
+    div.max(rem).max(add).max(mul).max(f_div).max(f_add).max(f_mul).max(f_sqrt)
```

Comment:
I'd put the bunch into an array and `.into_iter().max().unwrap()` to save typing.  Actually, these are all identical, why don't we *start* with something like

```rust
let binary_ops = [("i64", "div_s"), ("i64", "rem_s")]
```

?

And then just loop, rather than copy-paste.